### PR TITLE
Feature/cholera

### PR DESCRIPTION
**Pull request by caldwellst**
Put the cholera alerts back into the updated Mailchimp monitoring system now that CERF is updating the data.

### CODE REVIEW

**Comment by hannahker** [comment]
Let's also include updates to the GitBook docs if this is adding back a new public-facing indicator!

**Comment by caldwellst** [comment]
Not publicly facing at all. Basically, we can use the system to send emails to specifically tagged individuals in Mailchimp without having it in our subscription form! I can take y'all through it if you're interested!

File: src/indicators/who_cholera/update_cholera.R [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
 box::use(../../signals/generate_signals)
 
-# get the raw and wrangled data. The wrangled data is passed
-# to generate up the new alerts and campaigns
+first_run <- as.logical(Sys.getenv("FIRST_RUN", unset = FALSE))
+dry_run <- as.logical(Sys.getenv("HS_DRY_RUN", unset = TRUE))
+dry_run_filter <- if (dry_run) c("ETH", "SSD") else NULL
+indicator_id <- "who_cholera"
+
+hs_logger$configure_logger()
```
Comment (hannahker) [inline src/indicators/who_cholera/update_cholera.R:15]: Need to also import: `box::use(../../utils/hs_logger)`

**Comment by caldwellst** [comment]
Have done the first run successfully. Here's an example campaign: http://eepurl.com/iS8aO2#BEN. I think we can merge and start tracking to see how we go.

**Review by hannahker — APPROVED** [review]
Looks good to merge to me!

**Pull request merged**